Some minor code fixes

teward opened the pull request:
(1) While *extremely* unlikely, there is a tiny risk, if we change something about blacklist types and then forget to update `gitmanager.py`, that we'll have something blow up due to a no-value variable.  Let's have an "Else" case that will prevent that.

(2) Class methods tend to have `cls`, not `self`, in their arguments.

ArtOfCode- commented:
(2) is stylistic, and I *think* (not 100% but fairly sure) that `self` is the more widely used.

ArtOfCode- commented:
^ TIL. Most of our stuff is instance methods, so should be self.

teward commented:
@ArtOfCode- @AWegnerGitHub If that's the case, I don't think they should be tagged as `@classmethod` then.  In either case, I've reverted that specific commit, and fixed up other parts.

teward commented:
Note that I've pushed a commit up for autopull that will include an IDE suppression on that; I run everything unfortunately through an IDE, and it was whining every time I committed, so I've just suppressed (like I did with other code changes).

AWegnerGitHub commented:
I'm on mobile so I can't check everything right now, but...we do have some
class methods. Those can use 'cls'. Everything not decorated as a class
method should use self.

On Dec 27, 2016 6:25 PM, "Thomas Ward" <notifications@github.com> wrote:

> @ArtOfCode- <https://github.com/ArtOfCode-> @AWegnerGitHub
> <https://github.com/AWegnerGitHub> If that's the case, I don't think they
> should be tagged as @classmethod then. In either case, I've reverted that
> specific commit, and fixed up other parts.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Charcoal-SE/SmokeDetector/pull/397#issuecomment-269402514>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGKb5yMwPyZhvcH8XqF6VwmPqHcgYJ4cks5rMax3gaJpZM4LWjmw>
> .
>
